pkg/apis/agones/v1/gameserver.go

```diff
@@ -133,12 +133,20 @@ type GameServerSpec struct {
 	Ports []GameServerPort `json:"ports"`
 	// Health configures health checking
 	Health Health `json:"health,omitempty"`
-	// Scheduling strategy. Defaults to "Packed".
+	// Scheduling strategy. Defaults to "Packed"
 	Scheduling apis.SchedulingStrategy `json:"scheduling,omitempty"`
+	// Logging specifies the log level for gameserver sidecars
```

Comment:
Can you make this match the comment below, e.g. "Logging specifies the log levels for Agones system containers"?

Reply:
Will update in a moment to:
```
 Logging specifies log levels for Agones system containers
```